eve/workers/zenko.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "zenko-test-pod"
+spec:
+  containers:
+  - name: aggressor
```

Comment:
Que?

Reply:
copy-paste, left-over of sylvain PR, but don't what it mean either